.eslintrc

```diff
@@ -9,6 +9,12 @@
     "unicorn/prefer-module": "off",
 
     // TODO: This can be removed if we decide to add jsdoc to all functions
-    "valid-jsdoc": "off"
+    "valid-jsdoc": "off",
+    "no-await-in-loop": "off",
+    "unicorn/filename-case": "off",
+    "no-negated-condition": "off",
+    "no-else-return": "off",
+    "no-eq-null": "off",
+    "eqeqeq": ["error", "smart"]
```

Comment:
similar to negated checks https://eslint.org/docs/latest/rules/eqeqeq. we should be using `===` / `!==` or check specifically for `undefined` or `null` etc

Reply:
yes, I'm using the `eqeqeq` with the `smart` option. But it was conflicting with the `no-eq-null`, that's why I tuned it off